### PR TITLE
add colo code

### PR DESCRIPTION
**Pull request by tommyongib**

### CODE REVIEW

**Review by ssarkar-ib — APPROVED** [review]
LGTM

**Comment by tommyongib** [comment]
@hdoddikindi please review

File: colo/build/build.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+repo=infobloxcto
```
Comment (hdoddikindi) [inline colo/build/build.properties:1]: Is there a public repo for charts? To avoid storing credentials.

**Review by hdoddikindi — APPROVED** [review]
Looks good. Just a comment about using public repo for charts.

File: colo/build/build.properties (hunk 1 above)
Comment (tommyongib) [reply colo/build/build.properties:1]: The public repo will be this github repo. I'll add more another PR later to set it up.

**Pull request merged**